src/Symfony/Bundle/FrameworkBundle/Routing/RouteLoaderContainer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * @internal
+ *
+ * @deprecated since Symfony 4.4
+ */
+class RouteLoaderContainer implements ContainerInterface
```

Comment:
The name : "^RouteLoader" because it concerns the "service route loader" feature of the FWB. "Container$" because it's a container. 
In the FWB, because it would have no usage in the component.

Reply:
I'd suggest naming it `LegacyRouteLoaderContainer`